packages/forms/src/UIForm/fieldsets/Array/DefaultArrayTemplate.component.js

```diff
@@ -34,7 +34,7 @@ function DefaultArrayTemplate(props) {
 				className={classNames(theme['tf-array-add'], 'tf-array-add')}
 				bsStyle={'info'}
 				onClick={onAdd}
-				label={t('ARRAY_ADD_ELEMENT', { defaultValue: 'New Element' })}
+				label={schema.btnLabel || t('ARRAY_ADD_ELEMENT', { defaultValue: 'New Element' })}
```

Comment:
To pass custom properties, don't put them directly in uiSchema. There is a sort of convention that we use in forms package, add an option object, where you can add the properties you want.

Please add the entry in documentation (readme in array folder), and a storybook.